e2e_tests/conftest.py

```diff
@@ -45,19 +45,22 @@ def pytest_sessionstart(session: pytest.Session) -> None:
     session.config.cache.set("api_key", api_key)
     session.config.cache.set("team_slug", team_slug)
 
-    global datasets
     datasets = setup_tests(ConfigValues(server=server, api_key=api_key, team_slug=team_slug))
-
+    # pytest.datasets = datasets
+    setattr(pytest, "datasets", datasets)
+    # Set the environment variables for running CLI arguments
+    environ["DARWIN_BASE_URL"] = server
+    environ["DARWIN_TEAM"] = team_slug
```

Comment:
Generally, after setting ENV's you want to revert them, as this is change to global state.

Reply:
It's slightly misleading - it looks like a change to global state, but posix env vars, which these will be in most cases, are in one of two scopes, global or local - `os.environ` is changing the local (process) scope, at least in a posix-like system.

In windows, it might set it globally.

Obv, unsetting these wouldn't hurt, just adding context.